src/components/AppLogo.tsx

```diff
@@ -3,8 +3,7 @@ import styled from 'styled-components';
 
 import { COLORS } from '@theme';
 import { translateRaw } from '@translations';
-
-import mycLogo from '@assets/images/icn-myc.svg';
+import Icon from 'components/Icon';
```

Comment:
Please use the alias when importing a file `@components/Icon`.
It facilitates the projects maintainability.